### PR TITLE
[quant][graph] Add quantized batch_norm2d support to graph mode

### PR DESCRIPTION
**Pull request by supriyar**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #36622 [quant][graph] Graph mode quantization support for sigmoid
* #36552 [quant][graph] Add quantized batch_norm2d_relu to graph mode
* **#36549 [quant][graph] Add quantized batch_norm2d support to graph mode**
* #36548 [quant][graph] Add valueQuantizable function

Summary:

Test Plan:
python test_quantize_script.py

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D21048965](https://our.internmc.facebook.com/intern/diff/D21048965)

### CODE REVIEW

**Review by jerryzh168 — APPROVED** [review]
Looks good, please move the batchnorm change from previous PR to here as well

File: test/quantization/test_quantize_script.py [hunk 1]
```diff
@@ -1390,6 +1390,22 @@ def forward(self, x, y):
                    .check("quantized::cat") \
                    .run(m.graph_for(data, data))
 
+    def test_qbatch_norm(self):
+        class M(torch.nn.Module):
+            def __init__(self):
+                super(M, self).__init__()
+                self.bn = torch.nn.BatchNorm2d(3).to('cpu', torch.float)
```
Comment (jerryzh168) [inline test/quantization/test_quantize_script.py:1397]: I think `cpu` is default?
